lib/trailer/concern.rb

```diff
@@ -27,7 +28,7 @@ def trace_event(event, resource = nil, **tags, &block) # rubocop:disable Metrics
           end
         elsif resource.respond_to?(:to_h)
           # This handles other types of data, such as GraphQL input objects.
-          resource.to_h.stringify_keys.each do |key, value|
+          resource.to_h.transform_keys(&:to_s).each do |key, value|
             tags[key] ||= value if key.to_s.match?(Trailer.config.auto_tag_fields) || Trailer.config.tag_fields.include?(key)
```

Comment:
```suggestion
            tags[key] ||= value if key.match?(Trailer.config.auto_tag_fields) || Trailer.config.tag_fields.include?(key)
```

Reply:
PS: maybe here better to use a symbols too?

Reply:
Good catch 👍